packages/eslint-config-vtex/rules/style.js

```diff
@@ -134,19 +134,35 @@ module.exports = {
     // Require or disallow padding lines between statements
     // https://eslint.org/docs/rules/padding-line-between-statements
     'padding-line-between-statements': [
-      'error',
+      'warn',
       // empty lines after declarations
-      // { blankLine: 'always', prev: ['const', 'let', 'var'], next: '*' },
-      // {
-      //   blankLine: 'any',
-      //   prev: ['const', 'let', 'var'],
-      //   next: ['const', 'let', 'var'],
-      // },
+      {
+        blankLine: 'always',
+        prev: ['const', 'let', 'var'],
+        next: '*',
+      },
+      // allow to have none or one blank line between declarations
+      {
+        blankLine: 'any',
+        prev: ['const', 'let', 'var'],
+        next: ['const', 'let', 'var'],
+      },
       // empty lines before returns
-      // { blankLine: 'always', prev: '*', next: 'return' },
       {
         blankLine: 'always',
-        prev: ['function', 'class'],
+        prev: '*',
+        next: 'return',
+      },
+      // empty lines before returns
```

Comment:
Is this comment right?
Also, regarding next rule on line 157~161, will that require a blank line before/after `case` and `break`?
Maybe I'm failing on how to read it properly.

Reply:
Yeah! You can read it like: 

if you find any of `prev` followed by any of the `next`, add a blank line between them. 
```
blankLine: always
prev: *
next: return
```
Means `add a blank line between ANYTHING and a return statement`

```
blankLike: always
prev: case,break
next: case,break
```

Means `add a blank line between any combination of case,break followed by a case,break`

Reply:
But forgot to add a `default` to that last one hehe, thanks!

Reply:
wait, so something like
```c
case 'FOO':
case 'BAR':
```

will be warned?

Reply:
Yeah, but that would also result in an actual error: 

![image](https://user-images.githubusercontent.com/12702016/85455224-14526780-b574-11ea-92c0-c18a2c98f977.png)

We are currently discouraging switch-case fallthrough for precaution (it's also in the eslint:recommended ruleset)

Reply:
gotcha! 👍 
now it makes more sense, didn't know fallthrough was discouraged